modules/issue_tracker/php/NDB_Form_issue_tracker.class.inc

```diff
@@ -31,29 +31,22 @@ class NDB_Form_Issue_Tracker extends NDB_Form
 {
 
     /**
- * Entry point for issue_tracker/edit.
- *
- * @return void
- */
-    function edit()
+    * Override default behaviour, since the page is loaded from issueIndex.js
+    *
+    * @return void
+    * @access public
+    */
+    function display()
     {
-        $user    =& User::singleton();
-        $issueID = $_GET['issueID'];
-
-        if ((empty($issueID)
-            || !isset($issueID))
-            && !$user->hasPermission('issue_tracker_reporter')
-        ) {
-            header("HTTP/1.1 403 Forbidden");
-        }
+        return null;
```

Comment:
This seems dangerous.

Display is what NDB_Page calls to display the page to the user, I'm not sure what dragons will surface if it just returns null.